torch/csrc/api/include/torch/nn/functional/activation.h

```diff
@@ -244,9 +241,8 @@ inline Tensor rrelu(Tensor& input,
 }
 } // namespace detail
 
-inline Tensor rrelu(Tensor& input, RReLUFuncOptions options = {},
-                    bool training = false) {
-  return detail::rrelu(input, options.lower(), options.upper(), options.inplace(), training);
+inline Tensor rrelu(Tensor& input, RReLUFuncOptions options = {}) {
+  return detail::rrelu(input, options.lower(), options.upper(), options.inplace(), options.training());
```

Comment:
parameters have different order than in functional.py:
`def rrelu(input, lower=1. / 8, upper=1. / 3, training=False, inplace=False)`

Reply:
Thanks for the catch! Fixed